plugins/dell/fu-plugin-dell-esrt.c

```diff
@@ -20,57 +20,6 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
-#include "dell-wmi-smi.h"
-
-static gboolean
```

Comment:
Did you confirm we no longer need this code? I'd love to squash up what we've done so far and start splitting it into reviewable chunks.

Reply:
I haven't been able to test any of this on real hardware yet, but I am confident that we can do this approach of just letting libsmbios handle the SMI over WMI VS SMI over SMM.